### PR TITLE
3.6 doc updates

### PR DESCRIPTION
**Pull request by esethna**
Some 3.6 doc updates working through release PR spreadsheet

### CODE REVIEW

File: source/administration/config-settings.rst [hunk 1]
```diff
@@ -153,6 +153,10 @@ This button resets the configuration settings by reloading the settings from the
 
 The workflow for failover without downing the server is to change the database line in the config.json file, click **Reload Configuration from Disk** then click **Recycle Database Connections** in the Advanced > Database section.
 
+Purge All Caches
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+This button purges all the in-memory caches for things like sessions, accounts and channels. Deployments using High Availability will attempt to purge all the servers in the cluster. Purging the caches may adversly impact performance.
```
Comment (jasonblais) [inline source/administration/config-settings.rst]: `for things like sessions, accounts and channels`  --> should we consider using another term for `things`?

`An option to purge all in-memory caches, including for sessions, accounts and channels.`

Comment (jasonblais) [inline source/administration/config-settings.rst]: `adversly` --> `adversely`

File: source/administration/config-settings.rst [hunk 2]
```diff
@@ -223,6 +227,13 @@ Maximum number of channels per team, including both active and deleted channels.
 | This feature's ``config.json`` setting is ``"MaxChannelsPerTeam": 2000`` with whole number input.                                                                    |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Max Notifications Per Channel
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ 
+Maximum total number of users in a channel before users typing messages, @all, @here, and @channel no longer send notifications because of performance.
```
Comment (jasonblais) [inline source/administration/config-settings.rst]: Propose `... before @all, @here or @channel no longer send notifications to maximize performance.`

File: source/help/settings/team-settings.md [hunk 3]
```diff
@@ -17,6 +17,12 @@ Your **Team Name** is displayed on the sign-in page, and in the top of the left-
 
 You can enter a name up to 15 characters in length. The length of team names is limited to ensure readability.
 
+#### Team Description
+
+Your **Team Description** is displayed on the team selection screen and in the tooltip when hovering over the team name in the left-hand sidebar.
```
Comment (jasonblais) [inline source/help/settings/team-settings.md]: `team selection screen` --> `Team Selection page`

`left-hand sidebar` --> `team sidebar`

**Comment by esethna** [comment]
Thanks Jason, updated

File: source/help/settings/team-settings.md [hunk 4]
```diff
@@ -17,6 +17,12 @@ Your **Team Name** is displayed on the sign-in page, and in the top of the left-
 
 You can enter a name up to 15 characters in length. The length of team names is limited to ensure readability.
 
+#### Team Description
+
+Your **Team Description** is displayed on the Team Selection page and in the tooltip when hovering over the team name in the team sidebar.
```
Comment (lfbrock) [inline source/help/settings/team-settings.md]: Should we switch this to "join teams" page or something, now that we don't show that page on log in? 

Maybe "is displayed when viewing the list of teams available to join" or something like that

Comment (esethna) [inline source/help/settings/team-settings.md]: Makes sense. Updated

**Comment by lfbrock** [comment]
+1

**Comment by esethna** [comment]
Verified on docs.mattermost.com

**Pull request merged**